pcdsdevices/mpod_apalis.py

```diff
@@ -120,11 +124,38 @@ def set_current(self, current_number: float) -> None:
 
     @max_voltage.sub_value
     def _new_max_voltage(self, value: float, **kwargs) -> None:
-        """Set explicit limits on voltage for better error reporting."""
-        bounds = (0, value)
+        self._update_max_voltage_limits(value or 1)
+
+    def _update_max_voltage_limits(self, value: Optional[float] = None) -> None:
+        """
+        Set explicit symmetric limits on voltage for better error reporting.
+
+        Refers to the parent module if it exists, to determine the limit scaling
+
+        If not, defaults to setting limits to positive definite
+        """
+        # Hacky, we need to be careful not to call this without a value from
+        # inside a callback, probably
+        if value is None:
+            value = self.max_voltage.get()
```

Comment:
Probably it'd be best to avoid codepaths that require this.

Broken record, but the safest pattern is to cache all the values we need using monitor callbacks and only call the lowest callback if all the values are ready.